packages/utils/webpack-config-path.ts

```diff
@@ -0,0 +1,7 @@
+import * as findup from "findup-sync";
```

Comment:
Is there a reason why you created a specific file just for this function?

Reply:
Not really, what do you suggest where we put this function?

Reply:
I think it makes sense to rename the file `find-root` to `find-path` and we can merge this file into it and export all the path related function from there? 🤔

Reply:
We can put it in `is-local-path.ts` and rename it to `path-utils.ts`

Reply:
Alright, on it. 👍